### ISSUE TITLE
Resizing widgets is not available when in landscape

### ISSUE THREAD

**Issue by kugiigi** [pavelprosto94/uHome#5]
### Steps to Reproduce
1. Rotate device to landscape
2. Press and hold a widget

### Expected Results
Resize button shows on the widget

### Actual Results
Resize button is not displayed

**Comment by pavelprosto94** [comment]
I did it on purpose. This is due to the fact that at the moment when the screen is rotated, the widgets are also rotated 90 degrees. But at the same time, they remain in the same place.
And now it turns out that the Y coordinate corresponds to the X coordinate and vice versa. It is difficult to teach the program these rules. So I temporarily disabled this feature
In the future, I will teach the program to drag and drop widgets depending on the screen orientation.

**Comment by Keneda-ut** [comment]
> In the future, I will teach the program to drag and drop widgets depending on the screen orientation.

I was about to open an issue for asking a second config for landscape mode, so that user set manually widget places independently from the portrait one.
Maybe this way it would be more easy to implement ?

**Comment by pavelprosto94** [comment]
Yes! A very interesting idea! Instead of worrying about the reinstallation algorithm, let the user decide for himself where his widget should be!
